content/docs/other-guides/kustomize.md

```diff
@@ -0,0 +1,218 @@
++++
+title = "Configuring Kubeflow with kustomize"
+description = "The basics of Kubeflow configuration with kustomize"
+weight = 10
++++
+
+Kubeflow makes use of [kustomize](https://kustomize.io/) to help customize YAML
+configurations. 
+
+With kustomize, you can traverse a Kubernetes manifest to add, remove or update 
```

Comment:
Nit: Suggest adding a comma after "remove"

Reply:
Done. Thanks for reminding me about the serial thing. :)